index.js

```diff
@@ -5,9 +5,10 @@ var randomBytes = require('randombytes')
 var unorm = require('unorm')
 
 var DEFAULT_WORDLIST = require('./wordlists/en.json')
+var JAPANESE_WORDLIST = require('./wordlists/ja.json')
 
 function mnemonicToSeed(mnemonic, password) {
-  var mnemonicBuffer = new Buffer(mnemonic, 'utf8')
+  var mnemonicBuffer = new Buffer(unorm.nfkd(mnemonic), 'utf8')
   var saltBuffer = new Buffer(salt(password), 'utf8')
```

Comment:
salt has unorm inside it. So I didn't bother.

However, multiple runs through nfkd will always produce the same result, so maybe for consistency we could do that.

I think it's fine as is though.